tests/output/kml.py

```diff
@@ -28,35 +28,35 @@ def setUp(self):
   def testWriteHeader(self):
     """Tests the WriteHeader function."""
     expected_header = (
-        b'<?xml version="1.0" encoding="utf-8"?>'
-        b'<kml xmlns="http://www.opengis.net/kml/2.2"><Document>')
+        '<?xml version="1.0" encoding="utf-8"?>'
+        '<kml xmlns="http://www.opengis.net/kml/2.2"><Document>')
 
     self._output_module.WriteHeader()
 
-    header = self._output_writer.ReadOutput()
+    header = self._output_writer.ReadOutputString()
     self.assertEqual(header, expected_header)
 
   def testWriteFooter(self):
     """Tests the WriteFooter function."""
-    expected_footer = b'</Document></kml>'
+    expected_footer = '</Document></kml>'
 
     self._output_module.WriteFooter()
 
-    footer = self._output_writer.ReadOutput()
+    footer = self._output_writer.ReadOutputString()
     self.assertEqual(footer, expected_footer)
 
   def testWriteEventBody(self):
     """Tests the WriteEventBody function."""
 
     # Test event object without geo-location.
     self._output_module.WriteEventBody(self._event_object)
-    self.assertEqual(self._output_writer.ReadOutput(), b'')
+    self.assertEqual(self._output_writer.ReadOutputString(), '')
```

Comment:
suggestion: maybe break this up in
```
event_body = self._output_writer.ReadOutputString()
self.assertEqual(event_body, '')
```

so that it is more clear that we are testing event body

Reply:
Done